assets/css/admin/reports/_filter.scss

```diff
@@ -15,10 +15,228 @@
 		float: none;
 		flex-grow: 1;
 
+		.graph-option-section {
+			display: flex;
+			align-items: center;
+		}
+
+		.edd-date-range-picker[data-range='other'] {
+			.edd-graphs-date-options {
+				border-top-right-radius: 4px;
+				border-bottom-right-radius: 4px;
+			}
+
+			.edd-date-range-dates, .edd-date-range-relative-dates {
+				display: none;
+			}
+		}
+
 		.edd-date-range-options {
 			display: inline-block;
 			margin: 10px 0;
 		}
+
+		.edd-graphs-date-options {
+			border-top-right-radius: 0px;
+			border-bottom-right-radius: 0px;
+		}
+
+		.edd-date-range-dates {
+			display: flex;
+			align-items: center;
+			border: 1px solid #8c8f94;
+			border-left: none;
+			color: #2c3338;
+			padding: 4px 10px;
+			margin-left: -5px;
+			border-top-right-radius: 4px;
+			border-bottom-right-radius: 4px;
+			cursor: pointer;
+			&.hidden {
+				display: none;
+			}
+		}
+
+		.edd-date-main-icon {
+			margin-right: 4px;
+		}
+
+		.edd-date-range-selected-date {
+			display: inline-block;
+		}
+
+		.edd-date-range-relative-dates {
+			display: flex;
+			align-items: center;
+			margin-left: 10px;
+			&.hidden {
+				display: none;
+			}
+		}
+
+		.edd-date-range-selected-relative-date {
+			position: relative;
+			display: flex;
+			align-items: center;
+			border: 1px solid #8c8f94;
+			padding: 4px 2px 4px 6px;
+			color: #2c3338;
+			margin-left: 10px;
+			margin-right: 10px;
+			border-radius: 4px;
+			cursor: pointer;
+			.arrow-down {
+				width: 16px;
+				height: auto;
+				margin-left: 6px;
+				margin-top: 2px;
+				vertical-align: middle;
+			}
+			&.opened {
+				.edd-date-range-relative-dropdown {
+					display: block;
+				}
+			}
+		}
+
+		.edd-date-range-relative-dropdown {
+			position: absolute;
+			z-index: 99;
+			width: 420px;
+			left: 50%;
+			top: 100%;
+			margin-top: 10px;
+			transform: translateX( -50% );
+			background-color: #fff;
+			border: 1px solid #8c8f94;
+			border-radius: 4px;
+			-webkit-box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.25);
```

Comment:
I think webpack handles the prefixing--I'm not sure you need to have that in the .scss files.